src/data.jl

```diff
@@ -522,12 +522,16 @@ end
 Base.getindex(d::DensityOfStates, ind) = (d.energy[ind], d.dos[ind], d.int[ind])
 
 # Setting up for gaussian smearing of DOS curve
-# Fourier transformed gaussian for convolution in smearing
```

Comment:
In principle you should include a docstring. That's less important for methods that aren't exported, so don't worry about it.